lib/docopt.rb

```diff
@@ -621,31 +584,31 @@ def dump_patterns(pattern, indent=0)
       elsif pattern.class.ancestors.include?(ParentPattern)
         out << ws << pattern.class.name << "(\n"
         for p in pattern.children
-          out << dump_patterns(p, indent+1).rstrip << "\n"
+          out << dump_patterns(p, indent + 1).rstrip << "\n"
```

Comment:
We don't need to apply rubocop on external libraries

Reply:
Why not? It's a one time thing, and if it makes them more readable...

Reply:
I think an external library would be something in, say, the ruby path. I know this is from another project but it's just as much a chromebrew file as anything else.

Reply:
And requesting changes upstream is imposing our code style on another project, which I feel a bit tenuous about.